roles/lokole/templates/lokole.conf.j2

```diff
@@ -0,0 +1,27 @@
+# Root directory goes to Lokole web server
+
+ProxyRequests off
+ProxyPass {{ lokole_url }}/ unix:{{ lokole_domain_socket }}|http://{{ iiab_hostname }}.{{ iiab_domain }}/
+
+<Location {{ lokole_url }}/>
+  ProxyPassReverse /
+  ProxyHTMLEnable On
+  ProxyHTMLURLMap / {{ lokole_url }}/
+  RequestHeader unset Accept-Encoding
+</Location>
+
+# /static directory is stored on filesystem
+Alias {{ lokole_url }}/static {{ lokole_install_path }}
```

Comment:
As is http://box/lokole fails, and http://box/lokole/static shows an imperfect/incomplete page with error:
```
This page does not exist.
```
FWIW when line 14 is replaced by `Alias {{ lokole_url }} {{ lokole_install_path }}` and then `systemctl reload apache2` is run, it does see to bypass all the immediate errors, as a temporary workaround anyway?

i.e. this allowed me to test creating "email addresses" such as USERNAME@iiab-box.lokole.ca

_(But I don't want to interfere with the intentions of the person who wrote roles/lokole/templates/lokole.conf.j2 &mdash; @aidan-fitz can you perhaps ask them to explain more?)_